### ISSUE TITLE
Not Up to Date

### ISSUE THREAD

**Issue by sallyhaj** [ungoogled-software/ungoogled-chromium-debian#351]
Hello developers,
I am following the guide "Building a binary package" on the main page of the project, the problem is that, when I do so, it's compiling/building version "120.0.6099.216-1" not the latest version which supposed to be 130x.
How can I jump to the new version?

Thank you.

**Comment by PF4Public** [comment]
Unfortunately this repo is not actively maintained, but you may find this information useful: https://github.com/ungoogled-software/ungoogled-chromium-debian/pull/328

**Comment by iskunk** [comment]
To elaborate, there are currently two different ways of building ungoogled-chromium for Debian in this repo.

The main one, as documented in the top-level README file, is out of date. It is relatively laborious to maintain, and so not only has it not been maintained in a while, it is unlikely to be picked up again in the future.

The secondary one---soon to become the main one, hopefully (#349)---resides in the `convert/` subdirectory. That is a process that will "convert" a Debian `chromium` package into an equivalent `ungoogled-chromium` one. I maintain it, it doesn't need updating very often, and I can confirm that it will work for the current version as of this writing (130.0.6723.69).

**Comment by creeloper27** [comment]
> Unfortunately this repo is not actively maintained, but you may find this information useful: [#328](https://github.com/ungoogled-software/ungoogled-chromium-debian/pull/328)

Any chance of this repo getting revived? Having a debian package is very usefull, does someone know what is going on with the maintainers of this debian package and the future plans for it?

**Comment by PF4Public** [comment]
> what is going on with the maintainers

No one is actively maintaining this repository

**Comment by iskunk** [comment]
Hi @creeloper27, to elaborate a bit:

The `120.0.6099.216-1` version that you were looking at is ancient---we're on 135 as of this writing---and not safe to use anymore due to numerous security vulnerabilities. I would not touch it if I were you.

> Any chance of this repo getting revived? Having a debian package is very usefull, does someone know what is going on with the maintainers of this debian package and the future plans for it?

This repo is due to be revived once #349 passes review. That is to say, the "[convert](https://github.com/ungoogled-software/ungoogled-chromium-debian/tree/unified/convert)" procedure will become the official approach used by this repo.

Reviving this repo in its old form is not going to happen. The old process was quite labor-intensive, and duplicative of work that the Debian Chromium maintainer was already doing. The people who were driving this process before ran out of time/interest/energy for it long ago, and I would not even want them to return to it given that a more efficient process exists.

Believe me, I want to get official Debian builds going again. #349 is where that will happen.

(If you want Ubuntu builds, I can help you out there, but Debian is still at the DIY stage.)

**Comment by creeloper27** [comment]
Regarding the revive of this repo that is great news! Thanks for sharing :D

Also, regarding the version, yeah, I have just noticed the other day how ancient the version of ungoogled-chromium in the Debian Package is, I should have checked it before even trying to install it this package.

Regarding this and the fact that the [Main ungoogled-chromium README.md](https://github.com/ungoogled-software/ungoogled-chromium?tab=readme-ov-file#downloads) guides people to [This Repository's README.md](https://github.com/ungoogled-software/ungoogled-chromium-debian?tab=readme-ov-file#ungoogled-chromium-debian) there should probably be a message on top of it explaning the current state of This Repository and the Debian Package, idealy something more or less like this to be newby friendly maybe?

## ⚠️ The Debian Package is currently **VERY OUT OF DATE** ⚠️
### 🚧 A Revival of Debian Package is ongoing but it will take some time, see [#349](https://github.com/ungoogled-software/ungoogled-chromium-debian/pull/349) for the current progress, in the meanwhile feel free to use the [Flatpak version](https://flathub.org/apps/io.github.ungoogled_software.ungoogled_chromium) 

Would a PR just to update  [The README.md](https://github.com/ungoogled-software/ungoogled-chromium-debian?tab=readme-ov-file#ungoogled-chromium-debian) be overkill? I will just leave this suggestion here for now.

**Comment by PF4Public** [comment]
> there should probably be a message on top of it explaning the current state of This Repository and the Debian Package, idealy something more or less like this to be newby friendly maybe?

A cursory look at the last version and this issue should be enough for a reasonable person I guess.

**Comment by VA1DER** [comment]
The team that maintains the "xtradeb" PPA includes very recent Ungoogled-Chromium.  ppa:xtradeb/apps
This is for Ubuntu, but whatever they are doing to make recent builds should be applicable to vanilla Debian.

EDIT:  And I agree that the current state here needs BIG LOUD FLAGS.  One shouldn't have to worry that the officially suggested install mechanism for a project is going to put an ancient and unsafe version on your machine.

**Comment by VA1DER** [comment]
I have created a pull request for updating the opening readme to indicate this is out of date and not to use.   Anything else is **irresponsible**.  

@SugaryHull @iskunk @networkException @tpo @berkley4 @AssassinK786 @wrt54g that is aimed at you.  Someone with responsitory access, please action.

**Comment by iskunk** [comment]
I agree with the idea in principle. While I have some feedback for the PR, I would need someone else to approve it, because I'm the team that maintains u-c on XtraDeb :-]
